.changeset/unlucky-socks-explode.md

```diff
@@ -0,0 +1,5 @@
+---
+'@crowdstrike/glide-core': patch
+---
+
+Top-level JSDoc comments in components are no longer stripped out.
```

Comment:
Just a heads up. It's possible we hear from people who've mistyped something and are using Lit Analyzer.